src/styles/navbar.css

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Doppio+One&display=swap');
+
+.navbar{
+    height: 10vh;
+    width: 100%;
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-around;
+    font-family: 'Doppio One', 'sans-serif';
+    border-bottom: 1px solid orange;
```

Comment:
Why? lol

Reply:
This was the best way I could think of doing it. I'll find a better way to do it.